winpty/src/windows.rs

```diff
@@ -9,7 +9,8 @@ use winpty_sys::*;
 
 use widestring::WideCString;
 
-pub enum ErrorCodes {
+#[derive(Clone, Copy, Debug)]
```

Comment:
I usually try to do `Copy, Clone` instead of `Clone, Copy`. Because the `Clone` constraint is just necessary for silly reasons, the interesting bit is `Copy`. So that should be what the reader sees first, the clone part is implied anyways.